src/components/content/Resultat.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div class="reultat col-lg-7">
+    <a href="javascript:;" @click="$emit('reset')">
+      <svg class="icon-svg">
+        <use xlink:href="#arrow-left" /></svg
+      >{{ $t('vaelg-virksomhedsform.sporgsmaal.resultat.forfra') }}</a
+    >
+    <div>
+      <h2 class="h1 mb-8" data-testid="title">{{ $t(content.title) }}</h2>
+      <div data-testid="text" v-html="$t(content.text)" />
+    </div>
+    <button class="mt-8 button button-tertiary" @click="$emit('reset')">{{ $t('vaelg-virksomhedsform.sporgsmaal.resultat.forfra') }}</button>
+  </div>
+</template>
+
+<script lang="ts">
+import { GuideAnswer } from '../../enums/guideAnswer.enum';
+import { PropType } from 'vue';
+
+export default {
+  name: 'Resultat',
+
+  props: {
+    answers: {
+      type: Object as PropType<GuideAnswer[]>,
+      required: true
+    }
+  },
+
+  computed: {
+    content() {
+      switch (JSON.stringify(this.answers)) {
+        case JSON.stringify([GuideAnswer.FIRST, GuideAnswer.FIRST, GuideAnswer.FIRST]):
+          return {
+            title: 'vaelg-virksomhedsform.sporgsmaal.resultat.enkeltmand.title',
+            text: 'vaelg-virksomhedsform.sporgsmaal.resultat.enkeltmand.text'
+          };
+        case JSON.stringify([GuideAnswer.FIRST, GuideAnswer.FIRST, GuideAnswer.SECOND]):
```

Comment:
man kunne overveje at have en funktion (evt. lokal)  der tager type fx. "pmv" og returnerer objektet med de 2 tekstnøgler så slipper du for en del duplikation.

pseudo:
const transform = (key:string) => { title: 'vaelg-virksomhedsform.sporgsmaal.resultat.[KEY].title', text: 'vaelg-virksomhedsform.sporgsmaal.resultat.[KEY].text' }

Reply:
done